support-frontend/assets/pages/weekly-subscription-checkout/helpers/deliveryDays.ts

```diff
@@ -59,14 +64,54 @@ function getTierThreeDeliveryDate(today?: number) {
 	return result;
 }
 
-function getGuardianAdLiteDate(today?: number) {
-	const firstValidDate = addDays(new Date(today ?? new Date()), 15);
-	return firstValidDate;
-}
+const productDeliveryDate = (
+	productKey: ActiveProductKey,
+	paperProductOptions?: ActivePaperProductOptions,
+): Date | undefined => {
+	console.log(paperProductOptions);
+	switch (productKey) {
+		case 'GuardianAdLite':
+			return addDays(new Date(), 15);
```

Comment:
What does the delivery date mean for Ad Lite?

Reply:
Its a date used with the 'whatsNext' component, which will need specialist copy for the Observer Sunday in next PR. Hence putting it here (as we will need a special case for Observer Sunday Paper too). Does renaming function to `productDeliveryOrStartDate` work? Updated

Reply:
Ah yes that rename sounds great, thanks for explaining.